packages/appstore/src/modules/traders-hub/account-with-wallets.tsx

```diff
@@ -1,115 +1,31 @@
+    const { data, isLoading, error } = useWalletList();
+    const [selected_wallet, setSelectedWallet] = React.useState('');
 
-const AccountWithWallets = observer(({ show_test_wallets = false }: TProps) => {
-    const {
-        client: { accounts },
-    } = useStore();
+    if (isLoading) return <Loading is_fullscreen={false} />;
 
-    const wallet_test_accounts: typeof accounts[0][] = [
-        {
-            balance: 10415.24,
-            currency: 'USD',
-            landing_company_shortcode: 'svg',
-            is_virtual: 1,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 0,
-            currency: 'EUR',
-            landing_company_shortcode: 'malta',
-            is_virtual: 0,
-            loginid: 'MFW12345',
-        },
-        {
-            balance: 0,
-            currency: 'AUD',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 3476.21,
-            currency: 'USD',
-            landing_company_shortcode: 'malta',
-            is_virtual: 0,
-            loginid: 'MFW12345',
-        },
-        {
-            balance: 0,
-            currency: 'ETH',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 2.45,
-            currency: 'BTC',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 0,
-            currency: 'LTC',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 0,
-            currency: 'USDC',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 0,
-            currency: 'USDT',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 0,
-            currency: 'eUSDT',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 0,
-            currency: 'tUSDT',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-    ];
-
-    // TODO: delete 'wallet_test_accounts' after testing
-    const wallet_accounts = show_test_wallets
-        ? wallet_test_accounts
-        : Object.keys(accounts)
-              .filter(key => accounts[key]?.account_category === 'wallet')
-              .reduce((acc, cur) => {
-                  acc.push({ ...accounts[cur], loginid: cur });
-                  return acc;
-              }, [] as typeof accounts[0][]);
+    if (error) return <p>{JSON.stringify(error)}</p>;
```

Comment:
@thisyahlen-deriv Will be handled in another PR? 🤔

Reply:
Im not aware of any error handling for wallets for now. Will remove it